apps/emqx_management/src/emqx_mgmt_api_subscriptions.erl

```diff
@@ -187,8 +187,10 @@ format(WhichNode, {{Topic, _Subscriber}, Options}) ->
         maps:with([qos, nl, rap, rh], Options)
     ).
 
+get_topic(Topic, #{share := <<"$queue">> = Group}) ->
+    filename:join([Group, Topic]);
 get_topic(Topic, #{share := Group}) ->
-    emqx_topic:join([<<"$share">>, Group, Topic]);
+    filename:join([<<"$share">>, Group, Topic]);
```

Comment:
This is probably not what we want, i.e. `emqx_topic:join/1` is the proper way to do things here. Otherwise, the results will be different between Linux and Windows for example.